src/seedsigner/views/scan_views.py

```diff
@@ -133,6 +133,8 @@ def run(self):
                 from seedsigner.views.seed_views import AddressVerificationStartView
                 address = self.decoder.get_address()
                 (script_type, network) = self.decoder.get_address_type()
+                seed_num = self.seed_num
```

Comment:
Cannot always be assumed that object has .seed_num, sometimes user will scan an address first, afterwards should be prompted to load a seed.